source/options.tsx

```diff
@@ -130,6 +130,10 @@ async function init(): Promise<void> {
 			select<HTMLAnchorElement>('#personal-token-link')!.host = dropdown.value;
 		});
 	}
+
+	if(navigator.userAgent.indexOf('Firefox') > -1) {
+		select<HTMLBodyElement>('body')!.style.backgroundColor = '#201F24';
```

Comment:
I think it would be better to inject a `.is-firefox` class into the body so the CSS can stay in CSS and it's more reusable for the future.


Also: 
```suggestion
		document.body.style.backgroundColor = '#201F24';
```